pyxem/generators/calibration_generator.py

```diff
@@ -163,6 +165,52 @@ def get_elliptical_distortion(
 
         return affine
 
+    def get_cal_points(self, num_points, mask=None, show=False):
+        data = self.calibration_data.data
+        i_shape = np.shape(data)
+        flattened_array = data.flatten()
+        indexes = np.argsort(flattened_array)
+        if isinstance(flattened_array, np.ma.masked_array):
+            indexes = indexes[flattened_array.mask[indexes] == False]
+        if radius is not None:
+            center = [np.floor_divide(np.mean(indexes[-num_points:]), i_shape[1]),
+                      np.remainder(np.mean(indexes[-num_points:]), i_shape[1])]
+            print(center)
+        # take top 5000 points make sure exclude zero beam
+        cords = [np.floor_divide(indexes[-num_points:], i_shape[1]),
+                 np.remainder(indexes[-num_points:], i_shape[1])]  # [x axis (row),y axis (col)]
+
+    def get_amorphous_elliptical_distortion(
```

Comment:
Am I correct in thinking that this is for doing in sample calibrations, rather than our existing code which does gold ring patterns?

Reply:
Yea, this is just a bit of code that I had for calibrating the sample using an amorphous ring rather than a gold ring.  It probably deserves it's own pull request but just kind of got lumped into this one.